src/linter.js

```diff
@@ -235,7 +235,7 @@ export default class Linter {
               var manifestParser = new ManifestJSONParser(
                 json,
                 this.collector,
-                {selfHosted: this.config.selfHosted},
+                {selfHosted: this.config.selfHosted, files:this.io},
```

Comment:
`files` seems like the wrong variable name here; I'd expect an array of files, but this is an `IO` object.

There really isn't a great variable name I can think of, so I'd say rename this to at least `io` so it's easier to trace through.